### PR TITLE
Generate SRT from audio and export as SRT file to file

### PR DESCRIPTION
**Pull request by yujinqiu**

### CODE REVIEW

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+		DE081A952ABFC60E00E8CD63 /* Model.swift in Sources */ = {isa = PBXBuildFile; fileRef = DE081A942ABFC60E00E8CD63 /* Model.swift */; };
+		DE081A9A2ABFE9DA00E8CD63 /* tiny.en-encoder.int8.onnx in Resources */ = {isa = PBXBuildFile; fileRef = DE081A972ABFE9D900E8CD63 /* tiny.en-encoder.int8.onnx */; };
+		DE081A9B2ABFE9DA00E8CD63 /* tiny.en-decoder.int8.onnx in Resources */ = {isa = PBXBuildFile; fileRef = DE081A982ABFE9DA00E8CD63 /* tiny.en-decoder.int8.onnx */; };
+		DE081A9C2ABFE9DA00E8CD63 /* tiny.en-tokens.txt in Resources */ = {isa = PBXBuildFile; fileRef = DE081A992ABFE9DA00E8CD63 /* tiny.en-tokens.txt */; };
+		DE081AA92ABFEC8500E8CD63 /* tiny.en-decoder.int8.ort in Resources */ = {isa = PBXBuildFile; fileRef = DE081AA52ABFEC5800E8CD63 /* tiny.en-decoder.int8.ort */; };
+		DE081AAA2ABFEC8500E8CD63 /* tiny.en-encoder.int8.ort in Resources */ = {isa = PBXBuildFile; fileRef = DE081AA62ABFEC5800E8CD63 /* tiny.en-encoder.int8.ort */; };
+		DE081AAB2ABFEC8500E8CD63 /* silero_vad.onnx in Resources */ = {isa = PBXBuildFile; fileRef = DE081A962ABFE69400E8CD63 /* silero_vad.onnx */; };
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: Could you delete these  lines from this file?

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+		DE081A962ABFE69400E8CD63 /* silero_vad.onnx */ = {isa = PBXFileReference; lastKnownFileType = file; path = silero_vad.onnx; sourceTree = "<group>"; };
+		DE081A972ABFE9D900E8CD63 /* tiny.en-encoder.int8.onnx */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = "tiny.en-encoder.int8.onnx"; sourceTree = "<group>"; };
+		DE081A982ABFE9DA00E8CD63 /* tiny.en-decoder.int8.onnx */ = {isa = PBXFileReference; lastKnownFileType = file; path = "tiny.en-decoder.int8.onnx"; sourceTree = "<group>"; };
+		DE081A992ABFE9DA00E8CD63 /* tiny.en-tokens.txt */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = "tiny.en-tokens.txt"; sourceTree = "<group>"; };
+		DE081AA52ABFEC5800E8CD63 /* tiny.en-decoder.int8.ort */ = {isa = PBXFileReference; lastKnownFileType = file; path = "tiny.en-decoder.int8.ort"; sourceTree = "<group>"; };
+		DE081AA62ABFEC5800E8CD63 /* tiny.en-encoder.int8.ort */ = {isa = PBXFileReference; lastKnownFileType = file; path = "tiny.en-encoder.int8.ort"; sourceTree = "<group>"; };
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: And these lines.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+		DEA657132ABF19730066A81D /* SherpaOnnxSubtitle */ = {
+			isa = PBXGroup;
+			children = (
+				DE081AAC2ABFF30A00E8CD63 /* Info.plist */,
+				DE081AA52ABFEC5800E8CD63 /* tiny.en-decoder.int8.ort */,
+				DE081AA62ABFEC5800E8CD63 /* tiny.en-encoder.int8.ort */,
+				DE081A982ABFE9DA00E8CD63 /* tiny.en-decoder.int8.onnx */,
+				DE081A972ABFE9D900E8CD63 /* tiny.en-encoder.int8.onnx */,
+				DE081A992ABFE9DA00E8CD63 /* tiny.en-tokens.txt */,
+				DE081A962ABFE69400E8CD63 /* silero_vad.onnx */,
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: And these lines.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+/* Begin PBXResourcesBuildPhase section */
+		DEA6570F2ABF19730066A81D /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				DE081AA92ABFEC8500E8CD63 /* tiny.en-decoder.int8.ort in Resources */,
+				DE081AAA2ABFEC8500E8CD63 /* tiny.en-encoder.int8.ort in Resources */,
+				DE081AAB2ABFEC8500E8CD63 /* silero_vad.onnx in Resources */,
+				DEA6571C2ABF19740066A81D /* Preview Assets.xcassets in Resources */,
+				DE081A9A2ABFE9DA00E8CD63 /* tiny.en-encoder.int8.onnx in Resources */,
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: And these lines.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+				DEA657192ABF19740066A81D /* Assets.xcassets in Resources */,
+				DE081A9B2ABFE9DA00E8CD63 /* tiny.en-decoder.int8.onnx in Resources */,
+				DE081A9C2ABFE9DA00E8CD63 /* tiny.en-tokens.txt in Resources */,
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: And these lines.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/ContentView.swift [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+struct ContentView: View {
+    @StateObject var subtitleViewModel = SubtitleViewModel()
+
+    var body: some View {
+        VStack {
+            VStack {
+                Text("SherpaOnnxSubtitle")
+                    .font(.title)
+                VStack(alignment: .leading) {
+                    Text("Audio format should be **mono** channel and **1.6khz** sample rate")
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/ContentView.swift]:
```suggestion
                    Text("Audio format should be **mono** channel and **16khz** sample rate")
```

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/Models/Document.swift [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+//  Document.swift
+//  YPlayer
+//
+//  Created by knight on 2023/6/5.
+//
+
+import SwiftUI
+import UniformTypeIdentifiers
+
+// 只能定义
```
Comment (csukuangfj) [inline ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/Models/Document.swift]: Could you please remove non-English comments?

**Review by csukuangfj — COMMENTED** [review]
Thanks! Left some minor comments.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj (hunk 3 above)
Comment (yujinqiu) [reply ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle.xcodeproj/project.pbxproj]: delete ort file and keep onnx,  make error explicit.

File: ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/ContentView.swift (hunk 6 above)
Comment (yujinqiu) [reply ios-swiftui/SherpaOnnxSubtitle/SherpaOnnxSubtitle/ContentView.swift]: Done

**Comment by csukuangfj** [comment]
Thanks for your contribution!

**Pull request merged**